### DISCUSSION TITLE
Please keep including product/strategy/scrum master related new grad roles

### DISCUSSION THREAD

**Discussion by TreyDobbs** [ReaVNaiL/New-Grad-2024#49] [General]
Not all of us with CS degrees are looking to become a SWE, so I'd just like to throw out there that if anyone sees any product management, product analyst, strategy, or scrum master new grad/associate/junior level roles please feel free to throw them onto this running list.

**Comment by kcchawla85** [comment]
Hey,
I am interested for the role. How can I reach you ?

Regards

**Comment by imnischaygowda** [comment]
Thank you guys for the list. Can you guys also Data roles like Data Scientist, Data Engineer, Data Analyst etc., new grad/associate/junior/associate level roles to the running list.
